pkg/ksyms/ksyms.go

```diff
@@ -11,25 +11,15 @@ import (
 	"sort"
 	"strconv"
 	"strings"
-	"sync"
 
 	"github.com/cilium/tetragon/pkg/logger"
 	"github.com/cilium/tetragon/pkg/option"
 
 	lru "github.com/hashicorp/golang-lru/v2"
 )
 
-var (
-	kernelSymbols    *Ksyms
-	setKernelSymbols sync.Once
-)
-
 func KernelSymbols() (*Ksyms, error) {
-	var err error
-	setKernelSymbols.Do(func() {
-		kernelSymbols, err = NewKsyms(option.Config.ProcFS)
-	})
-	return kernelSymbols, err
+	return NewKsyms(option.Config.ProcFS)
 }
```

Comment:
nit: Could we add a comment there? it might be a little bit confusing why we had this function in the first place.